cypress/integration/jsGrowthTableDOM.spec.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+Cypress.Commands.add('clickButton',
+(selector, times) => {
+  for (let n = 0; n < times; n++) {
+    cy.get(selector).click( {force: true} );
+  }
+});
+
+describe('Table', () => {
+  beforeEach(() => {
+    cy.visit('');
+    cy.get('tbody > tr').as('rows');
+    cy.get('tbody :nth-child(1) :nth-child(n)').as('columns');
+  });
+
+  it('should have 4 rows by default', () => {
+    cy.get('@rows').should('have.length', 4);
+  });
+
+  it('should add 1 row by click', () => {
+    cy.get('.append-row').click();
+    cy.get('@rows').should('have.length', 5);
+  });
+
+  it('should have 4 columns by default', () => {
+    cy.get('@columns').should('have.length', 4);
+  });
+
+  it('should add 1 column by click', () => {
+    cy.get('.append-column').click();
+    cy.get('@columns').should('have.length', 5);
+  });
+
+  it('should have max 10 columns', () => {
+    cy.clickButton('.append-column', 7);
```

Comment:
It might be hard for the reader to understand what is magic number 7. In such cases it makes sense to add comments.

```suggestion
    // 4 columns by default and click 7 times to add 7 columns
    cy.clickButton('.append-column', 7);
```